--- concedit.cc	2013-05-10 15:00:46.000000000 +0200
+++ concedit-new.cc	2014-05-14 11:39:37.162108991 +0200
@@ -198,6 +198,12 @@
     coll_count [collnum - 1] = 0;
 }
 
+int custom_rand(int i) 
+{ 
+    srand(1298131);
+    return rand() % i;
+}
+
 void Concordance::shuffle()
 {
     sync();
@@ -206,7 +212,7 @@
         for (ConcIndex i=0; i < size(); i++)
             (*view)[i] = i;
     }
-    random_shuffle(view->begin(), view->end());
+    random_shuffle(view->begin(), view->end(), custom_rand);
 }
 
 void Concordance::switch_aligned (const char *corpname)
